chips/earlgrey/src/pinmux_config.rs

```diff
@@ -0,0 +1,40 @@
+// Licensed under the Apache License, Version 2.0 or the MIT License.
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// Copyright Tock Contributors 2022.
```

Comment:
Check the year on the copyright -- this should probably be 2023.

Reply:
Updated.